cmd/run.go

```diff
@@ -50,6 +52,14 @@ func addModelsFlag(cmd *cobra.Command) {
 	}
 }
 
+func addFailOnNotFoundFlag(cmd *cobra.Command) {
+	cmd.Flags().BoolVarP(&FailOnNotFound, "fail-on-not-found", "f", true, "Fail if given models are not found")
+	err := cmd.RegisterFlagCompletionFunc("fail-on-not-found", completeModelFilterFn)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
This will add the shell auto complete for this flag, which would result in it trying to auto complete model names for the true/false value
```suggestion
```